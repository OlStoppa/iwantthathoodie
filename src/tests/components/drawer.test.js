import React from "react"
import {render, cleanup } from "@testing-library/react"
import "@testing-library/jest-dom/extend-expect"
import Drawer from "../../components/drawer"


afterEach(cleanup);

test("renders", () => {
    const { asFragment } = render(<Drawer />);
    expect(asFragment()).toMatchSnapshot()
})

test("different props add classes for open or closed", () => {
    const { container, rerender } = render(<Drawer isOpen={true} /> );
    expect(container.firstChild).toHaveClass('drawer open');

    rerender(<Drawer isOpen={false} /> );
    expect(container.firstChild).toHaveClass('drawer');
})