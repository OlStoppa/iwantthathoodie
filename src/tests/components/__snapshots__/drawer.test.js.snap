// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="drawer"
  >
    <div
      class="mobile-nav-container"
    >
      <ul>
        <a
          href="/contact"
        >
          <li>
            Contact Us
          </li>
        </a>
      </ul>
    </div>
  </div>
</DocumentFragment>
`;
